### PR TITLE
Allows dittybopper to be deployed in user-specified namespaces

### PR DESCRIPTION
**Pull request by krishvoor**
### Description
Allow dittydopper to be deployed in user-specific namespaces.

Signed-off-by: Krishna Harsha Voora <krvoora@redhat.com>

### CODE REVIEW

**Comment by krishvoor** [comment]
@smalleni @mukrishn ptal

**Comment by mukrishn** [comment]
I would rather default it to `dittybopper` ns and make it overrid'able if they need to be on user-specified ns, but is there any benefit on making this on a different ns ?

**Comment by krishvoor** [comment]
Yes, the default is always `dittybopper` ns and the existing logic is also inline to be [overridden](https://github.com/cloud-bulldozer/performance-dashboards/blob/master/dittybopper/deploy.sh#L58-L60) w/ a user-specified ns, however, the deployment fails as below given.

This PR appreciates the inline logic and allows it to be deployed in `non-dittybopper` ns as well

```
Using k8s command: oc
Using namespace: harsha
Using default grafana password: admin

Getting environment vars...
Prometheus URL is: https://prometheus-k8s-openshift-monitoring.apps.xxxxxx/
Prometheus bearer token collected.

Creating namespace...
namespace/harsha created

Deploying Grafana...
Error from server (NotFound): error when creating "STDIN": namespaces "dittybopper" not found
Error from server (NotFound): error when creating "STDIN": namespaces "dittybopper" not found
Error from server (NotFound): error when creating "STDIN": namespaces "dittybopper" not found
Error from server (NotFound): error when creating "STDIN": namespaces "dittybopper" not found
Error from server (NotFound): error when creating "STDIN": namespaces "dittybopper" not found
[root@krvoora dittybopper]#
```

**Comment by mukrishn** [comment]
Ahh got it, it was there already but templates have hard coded it. Also make sure you [apply](https://github.com/cloud-bulldozer/performance-dashboards/blob/master/dittybopper/deploy.sh#L138) this on the user specified project.

**Comment by krishvoor** [comment]
>Also make sure you [apply](https://github.com/cloud-bulldozer/performance-dashboards/blob/master/dittybopper/deploy.sh#L138) this on the user specified project.

Thanks for the pointer, I pushed those changes as of now, can you ptal?

**Comment by krishvoor** [comment]
[user_ns.log](https://github.com/cloud-bulldozer/performance-dashboards/files/9251569/user_ns.log)
[default_ns.log](https://github.com/cloud-bulldozer/performance-dashboards/files/9251572/default_ns.log)

File: dittybopper/templates/k8s-dittybopper.yaml.template [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: dittybopper
-  namespace: dittybopper
+  namespace: dittybopper
```
Comment (mukrishn) [inline dittybopper/templates/k8s-dittybopper.yaml.template:5]: Lets make this also dynamic.

Comment (krishvoor) [reply dittybopper/templates/k8s-dittybopper.yaml.template]: Done

**Review by rsevilla87 — APPROVED** [review]
ok, so the -n argument was already there, but we weren't using it.
lgtm!

**Pull request merged**